vips/vip-391/bsctestnet.ts

```diff
@@ -0,0 +1,136 @@
+export const USDC_PRIME_CONVERTER = "0x511a559a699cBd665546a1F75908f7E9454Bfc67";
+export const WBTC_PRIME_CONVERTER = "0x8a3937F27921e859db3FDA05729CbCea8cfd82AE";
+export const WETH_PRIME_CONVERTER = "0x274a834eFFA8D5479502dD6e78925Bc04ae82B46";
+export const XVS_VAULT_CONVERTER = "0xc203bfA9dCB0B5fEC510Db644A494Ff7f4968ed2";
+export const BaseAssets = [
+  "0x8d412FD0bc5d826615065B931171Eed10F5AF266", // USDT USDTPrimeConverter BaseAsset
+  "0x772d68929655ce7234C8C94256526ddA66Ef641E", // USDC USDCPrimeConverter BaseAsset
+  "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b", // WBTC WBTCPrimeConverter BaseAsset
+  "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9", // WETH WETHPrimeConverter BaseAsset
+  "0x66ebd019E86e0af5f228a0439EBB33f045CBe63E", // XVS XVSPrimeConverter BaseAsset
+];
+const CONVERSION_INCENTIVE = parseUnits("0.0001", 18);
+
+const vip391 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-391",
+    description: `### Description`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      // Configure Oracle
+      {
+        target: MOCK_ACCOUNTANT,
+        signature: "setRate(uint256)",
+        params: [RATE],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [eBTC, [ACCOUNTANT_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero], [true, false, false]],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Add Market
+      {
+        target: eBTC,
+        signature: "faucet(uint256)",
+        params: [INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: eBTC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: eBTC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: veBTC,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [[veBTC, CF, LT, INITIAL_SUPPLY, sepolia.VTREASURY, SUPPLY_CAP, BORROW_CAP]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: veBTC,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.01", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
```

Comment:
We don't need to update this value because this market will be added to the Core pool, where the `protocolSeizeShare` value should be 0.05 (as it's already set in the contract)

Reply:
```suggestion
```